src/kudu/gutil/template_util.h

```diff
@@ -46,8 +46,10 @@
 // Both of these outcomes means that we may be able to directly replace
 // some of these with boost equivalents.
 //
-#ifndef BASE_TEMPLATE_UTIL_H_
-#define BASE_TEMPLATE_UTIL_H_
+
+// These #defines collide with sparsehash
+#ifndef KUDU_BASE_TEMPLATE_UTIL_H_
```

Comment:
This is a clever mitigation :) Will this stay for good or is it temporary setup for milestone 1?

Reply:
Happy to fix this upstream to use pragma once

Reply:
That would be great, @toddlipcon